### ISSUE TITLE
Add tick to conda-forge

### ISSUE THREAD

**Issue by MarcosCarreira** [X-DataInitiative/tick#458]
Those of us who use the Anaconda distribution would benefit from managing tick within the conda environment: [https://conda-forge.org/#about](https://conda-forge.org/#about)

**Comment by sumau** [comment]
@MarcosCarreira  You can pip install pip libraries into conda environments :
https://realpython.com/python-windows-machine-learning-setup/#using-pip-inside-conda-environments

**Comment by MarcosCarreira** [comment]
Yes, but adding it to the conda environment makes it easier to check for upgrades and it would reach a wider base of users if it became part of the distribution.
